erpnext/projects/doctype/project/project.py

```diff
@@ -1091,6 +1098,40 @@ def get_item_groups_subtree(self, item_group):
 
 		return self._item_group_subtree[item_group]
 
+	def get_vehicle_in_workshop(self):
+			vehicle_service_receipts = None
+			vehicle_gate_passes = None
+
+			if self.get('applies_to_vehicle'):
+				vehicle_service_receipts = frappe.db.get_all("Vehicle Service Receipt",
+					{"project": self.name, "vehicle": self.applies_to_vehicle, "docstatus": 1},
+					['name', 'posting_date', 'posting_time'],
+					order_by="posting_date, posting_time, creation")
+
+				vehicle_gate_passes = frappe.db.get_all("Vehicle Gate Pass",
+					{
+						"project": self.name,
+						"vehicle": self.applies_to_vehicle,
+						"docstatus": 1,
+						"purpose": "Service - Vehicle Delivery"
+					},
+					['name', 'posting_date', 'posting_time'],
+					order_by="posting_date, posting_time, creation")
+
+			vehicle_service_receipt = frappe._dict()
+			vehicle_gate_pass = frappe._dict()
+
+			if vehicle_service_receipts:
+				vehicle_service_receipt = vehicle_service_receipts[0]
+
+			if vehicle_gate_passes:
+				vehicle_gate_pass = vehicle_gate_passes[-1]
+
+			if vehicle_service_receipt.name and not vehicle_gate_pass.name:
+				return True
+
+			return False
+
```

Comment:
seems like duplicate functionality for `vehicle_status`. Can we rely on `vehicle_status` instead of setting onload?